lib/Common/Common/DateUtilities.cpp

```diff
@@ -68,7 +68,7 @@ namespace Js
         _u("PDT")
     };
 
-
+#ifdef INTL_WINGLOB
     //
```

Comment:
I noticed `DateUtilities::ES5DateToWinRTDate ` whilst looking for references to WinRTDate - this doesn't actually use the WinRTDate type however it is only used within `EntryIntl_FormatDateTime` and within that function it is guarded by an `#ifdef INTL_WINGLOB` - hence wrapping it with the same ifdef.

Side note the flag INTL_WINGLOB is to specify using the Windows built in Globalisation library instead of ICU - there is a lot code behind it in places that I would prefer to remove in the long term and just assume ICU will be used in all builds.